src/rally-content.ts

```diff
@@ -80,17 +80,27 @@ function handleBackgroundEvents(message: { type: WebMessages, data: {}; }, sende
   }
 }
 
-// Listen for a web-check message from the website.
-// @ts-ignore
-window.addEventListener(WebMessages.WebCheck, e => handlePageEvents(e));
+let isStarted = false;
 
-// Listen for a complete-signup-response message from the website.
-// @ts-ignore
-window.addEventListener(WebMessages.CompleteSignupResponse, e => handlePageEvents(e));
+export function startContentScript() {
+  if (isStarted) {
+    return;
+  }
+
+  isStarted = true;
+
+  // Listen for a web-check message from the website.
+  // @ts-ignore
+  window.addEventListener(WebMessages.WebCheck, e => handlePageEvents(e));
```

Comment:
```suggestion
  window.addEventListener(WebMessages.WebCheck, (e: CustomEvent) => handlePageEvents(e));
```

Reply:
Removed dist files. Once we have the NPM credentials we can add npm publish?